components/fs/nfs/tcp.c

```diff
@@ -357,11 +359,12 @@ err_t socket_connected(void *arg, struct tcp_pcb *tpcb, err_t err)
     assert(socket->state == socket_state_connecting);
 
     socket->state = socket_state_connected;
-    
+
     tcp_sent(tpcb, socket_sent_callback);
     tcp_recv(tpcb, socket_recv_callback);
 
     tpcb->so_options |= SOF_KEEPALIVE;
+    socket->sock_tpcb->keep_intvl = 1000;
```

Comment:
Can we have something like `#define LWIP_KEEPALIVE_INTERVAL 1000` in `lwipopts.sh`?

Reply:
And also a comment next to the define that says it's 10s etc.